apps/dotcom/sync-worker/src/postgres.ts

```diff
@@ -39,7 +14,7 @@ export function createPostgresConnectionPool(env: Environment, name: string) {
 			connectionString: env.BOTCOM_POSTGRES_POOLED_CONNECTION_STRING,
 			application_name: name,
 			idleTimeoutMillis: 10_000,
-			max: 450,
```

Comment:
i think this should be 1? since it's not a shared connection pool, it's just one pool per durable object, and really we should only need one connection per durable object?

Reply:
looks like kysely's postgres dialect straight up requires a connection pool, no option for a single client. 

The only potential downside might be accidentally blocking some operations? e.g. the replicator fetches guest file subscriptions, wouldn't want those to block each other. I'll bump this up to 100 for the replicator, and keep it at 1 for other things that don't serve multiple clients.